src/axom/sidre/core/Group.hpp

```diff
@@ -1335,6 +1368,13 @@ class Group
    */
   void printTree(const int nlevels, std::ostream& os = std::cout) const;
 
+  /*!
+    * \brief Print in a way that won't crash for non-host data.
+    *
+    * If data is not host-accessible, print the pointer and a comment.
+    */
```

Comment:
Group has several print functions. 

Please update the docs here to mention what this function is printing, e.g. is it hierarchical? does it print the views, etc...

Looking at the implementation, it seems that it's hierarchical, so perhaps it should be named `hostPrintTree` ?